src/FluentValidation.AspNetCore/FluentValidationModelValidatorProvider.cs

```diff
@@ -37,22 +37,25 @@ namespace FluentValidation.AspNetCore;
 public class FluentValidationModelValidatorProvider : IModelValidatorProvider {
 	private readonly bool _implicitValidationEnabled;
 	private readonly bool _implicitRootCollectionElementValidationEnabled;
+	private readonly Func<Type, bool> _filter;
 
 	public FluentValidationModelValidatorProvider(bool implicitValidationEnabled)
-		: this(implicitValidationEnabled, false) {
+		: this(implicitValidationEnabled, false, default) {
 	}
 
 	public FluentValidationModelValidatorProvider(
 		bool implicitValidationEnabled,
-		bool implicitRootCollectionElementValidationEnabled) {
+		bool implicitRootCollectionElementValidationEnabled,
+		Func<Type, bool> filter) {

```

Comment:
Adding a new parameter here is an API breaking change, so will need to add an additional constructor for backwards compatibility that has just the 2 bool parameters.

Reply:
Sorry I missed that.  I added a 2-param constructor for backward compatibility.  Thanks!